packages/hurl_core/src/parser/base64.rs

```diff
@@ -32,7 +32,7 @@ https://en.wikipedia.org/wiki/Base64
 Test padding/no-padding
 
 Encoded
-YW55IGNhcm5hbCBwbGVhcw==		any carnal pleas   # [97, 110, 121, 32, 99, 97, 114, 110, 97, 108, 32, 112, 108, 101, 97, 115]
```

Comment:
`any carnal pleas` should be left as it as it's the base64 decoded string of  `YW55IGNhcm5hbCBwbGVhcw==`